app/models/enterprise.rb

```diff
@@ -198,10 +188,6 @@ def self.distinct_count
     count(distinct: true)
   end
 
-  def activated?
-    confirmed_at.present? && sells != 'unspecified'
```

Comment:
@oeoeaio I considered that as well, but won't all owners be confirmed if they have to confirm their email before registering an enterprise or logging in to the backend?